tests/security/cc/setup_net_test_env.pm

```diff
@@ -1,9 +1,9 @@
 # SUSE's openQA tests
 #
-# Copyright 2021 SUSE LLC
+# Copyright 2021-2022 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 #
-# Summary: Run 'netfilter' test case of 'audit-test' test suite
+# Summary: Setup network configuration for netfilter and netfilebt test suites
 # Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
 # Tags: poo#96540 poo#99441, poo#110157
```

Comment:
One minor comment, add the ticket number here, others lgtm, great job!!

Reply:
s/suites/cases/